integration_tests/__tests__/__snapshots__/failures-test.js.snap

```diff
@@ -62,7 +62,7 @@ Object {
     Expected value to be truthy, instead received
       false
       
-      at Object.throws (__tests__/assertion-count-test.js:14:17)
+      at throws (__tests__/assertion-count-test.js:14:17)
```

Comment:
True, it's not strictly necessary to display Object here but the general point is about instances of a function/class that should show the correct name. Can you make sure there is a test that proves this?